internal/vds/vds_test.go

```diff
@@ -689,19 +546,16 @@ func TestFenceInterpolationDefaultIsNearest(t *testing.T) {
 }
 
```

Comment:
The one above is fine, if you don't want to change it, but not sure if you wanted to but didn't go through the whole file.